src/Wellcome.Dds/Wellcome.Dds.Dashboard/Controllers/GoobiCallController.cs

```diff
@@ -56,9 +57,18 @@ public async Task<ActionResult> GoobiCall(string id)
 
         public async Task<ActionResult> Create(string id)
         {
+            var opts = Request.Query["options"].ToString();
```

Comment:
I think this could be added as a method param `int? options` which would save manual parsing. Might need `[FromQuery]` but I don't think so.